backend/open_api_spec.yml

```diff
@@ -0,0 +1,829 @@
+        "401":
+          $ref: "#/components/responses/401"
+    post:
+      description: Create a new Collection
+      security:
+        - cxguserCookie: []
+      requestBody:
+        content:
+          application/json:
+            $ref: "#/components/schemas/collection_metadata"
+      responses:
+        "201":
+          description: Created
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/collection_uuid"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/collections/{collection_id}:
+    delete:
+      description: Delete a private Collection. Delete an existing revision for a public Collection.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              type: string
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    get:
+      description: Fetch Collection metadata and associated Datasets, public and private.
+      security:
+        - cxgUserCookieLenient: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              $ref: "#/components/schemas/collection"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    put:
+      description: Start a revision. Update any included Collection metadata fields.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+      requestBody:
+        content:
+          application/json:
+            $ref: "#/components/schemas/collection_metadata"
+      responses:
+        "204":
+          $ref: "#/components/responses/204"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+
+  /curation/collections/{collection_id}/datasets:
+    delete:
+      description: >
+        Delete a private Dataset. For a public Dataset, start a revision if
+        one does not exist, then delete the Dataset from the revision.
+      security:
+        - cxguserCookie: []
+      parameters:
+        - $ref: "#/components/parameters/path_collection_uuid"
+        - $ref: "#/components/parameters/optional_query_curator_tag"
+        - $ref: "#/components/parameters/optional_query_dataset_uuid"
+        - $ref: "#/components/parameters/query_visibility"
+      responses:
+        "204":
+          $ref: "#/components/responses/204"
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
```

Comment:
Remove this. We always return 202 on delete whether the file is there or not.

Reply:
What about for the super curator, who has permission to view everything?

Reply:
404